### DISCUSSION TITLE
BurgerPanel 1.1.4

### DISCUSSION THREAD

**Discussion by TheBlueBurger** [TheBlueBurger/BurgerPanel#21] [Announcements]
Add built in plugin downloader, add file uploader, add file downloader, fix bug causing server.properties getting nuked

<hr /><em>This discussion was created from the release <a href='https://github.com/TheBlueBurger/BurgerPanel/releases/tag/'>BurgerPanel 1.1.4</a>.</em>

**Comment by TheRedXD** [comment]
POG
